src/nft/components/details/detailsV2/shared.ts

```diff
@@ -7,3 +7,8 @@ export const containerStyles = css`
   width: 100%;
   align-self: flex-start;
 `
+
+export function getBubbleText(num: number) {
```

Comment:
don't really think this belongs in the shared folder considering its only going to be used in one file

Reply:
good callout, I previously hadn't made the bubble text part of tabbedComponent so it was more distributed. Given it's only used in one location I've deprecated it as a helper fn